### PR TITLE
[CA-1853] Remove GPAlloc from leonardo test code

### PR DESCRIPTION
**Pull request by ehigham**
RR: https://broadworkbench.atlassian.net/browse/CA-1853
Update test code for changes to workbench-libs that removed support for
GPAlloc.

TODO: need to find a billing account to use for tests... Not sure if I can hard-code this or edit the config for leonardo tests to include this.

---
Have you read [CONTRIBUTING.md](https://github.com/DataBiosphere/leonardo/blob/develop/CONTRIBUTING.md) lately? If not, do that first.

I, the developer opening this PR, do solemnly pinky swear that:

- [ ] I've documented my API changes in Swagger

In all cases:

- [ ] Get a thumbsworth of review and PO signoff if necessary
- [ ] Verify all tests go green (you can re-run automation tests with a comment saying `jenkins retest`
- [ ] Run the automation tests multiple times in parallel to weed out instability if applicable via a comment saying `multi-test`
- [ ] Squash and merge; Delete your branch after this
- [ ] Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

**Comment by ehigham** [comment]
jenkins rerun

**Comment by ehigham** [comment]
jenkins retry

**Comment by Qi77Qi** [comment]
jenkins retest

File: automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/LeonardoSuite.scala [hunk 1]
```diff
@@ -1,25 +1,27 @@
...
 import org.broadinstitute.dsde.workbench.model.google.GoogleProject
-import org.broadinstitute.dsde.workbench.service.{BillingProject, Orchestration, Rawls}
+import org.broadinstitute.dsde.workbench.service.BillingProject.BillingProjectRole
+import org.broadinstitute.dsde.workbench.service.{Orchestration, Rawls}
 import org.scalatest._
 import org.scalatest.freespec.FixtureAnyFreeSpecLike
-import java.util.UUID
 
-trait GPAllocFixtureSpec extends FixtureAnyFreeSpecLike with Retries with LazyLogging {
+trait BillingProjectFixtureSpec extends FixtureAnyFreeSpecLike with Retries with LazyLogging {
```
Comment (jdcanas) [inline automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/LeonardoSuite.scala:24]: I think there's a few mentions of this trait in the readme, would you mind ctrl + f for `GPAllocFixtureSpec` and `GPAllocUtils` and ensuring the documentation stays up to date?

**Review by Qi77Qi — APPROVED** [review]
thanks @ehigham ! 🎉

**Comment by Qi77Qi** [comment]
multi-test

File: automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/LeonardoSuite.scala (hunk 1 above)
Comment (ehigham) [reply automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/LeonardoSuite.scala:24]: Thanks. I also removed all reference to "gpalloc" in names in automation. I the proejcts used in unit tests look real (though I haven't verifiied) so change those. Does that sound OK?

Comment (jdcanas) [reply automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/LeonardoSuite.scala:24]: Hmm... not sure exactly which projects you are referring to. It seems like any hardcoded projects in the tests are purely for json  decoding/encoding testing. Shouldn't be much to update besides any comments

**Review by jdcanas — APPROVED** [review]

**Pull request merged**